src/main/webapp/i18n/en/multiRecordView.json

```diff
@@ -15,6 +15,7 @@
       "error": "Error occurred while removing the match!"
     },
     "yourData": "Your data",
+    "clickHereForWebsite": "Click the icon for website",
```

Comment:
This sounds a bit weird. What icon, the question mark one? What do you mean by "for website"? Will it be copied to clipboard or will user be redirected to it?

Reply:
User will be redirected. Would 'Go to website' be better?